RELEASE_NOTES.md

```diff
@@ -2,85 +2,58 @@
 
-  * `power_bounds` is replaced by `power_status`
-  * The `power_status` method streams objects containing:
-    + bounds adjusted to the actor's priorities
-    + the latest target power for the set of batteries
-    + the results from the power distributor for the last request
+  * `logical_meter()`: The `*_consumption()` and `*_production()` methods were removed. You should use the new `consumption` and `production` formula operators instead.
 
-- Move `microgrid.ComponentGraph` class to `microgrid.component_graph.ComponentGraph`, exposing only the high level interface functions through the `microgrid` package.
+    For example:
 
-- An actor that is crashing will no longer instantly restart but induce an artificial delay to avoid potential spam-restarting.
+    ```python
+    # Old:
+    pv_consumption = logical_meter.pv_consumption_power()
+    production = (logical_meter.pv_production_power() + logical_meter.chp_production_power()).build()
+    # New:
+    pv_consumption = logical_meter.pv_power().consumption().build()
+    production = (logical_meter.pv_power().production() + logical_meter.chp_power().production()).build()
+    ```
 
 ## New Features
 
-- New and improved documentation.
-
-  * A new *User Guide* section was added, with:
-
-    + A glossary.
-    + An introduction to actors.
-
-  * A new *Tutorials* section was added, with:
-
-    + A getting started tutorial.
-
-- In `OrderedRingBuffer`:
-  - Rename `datetime_to_index` to `to_internal_index` to avoid confusion between the internal index and the external index.
-  - Add `index_to_datetime` method to convert external index to corresponding datetime.
-  - Remove `__setitem__` method to enforce usage of dedicated `update` method only.
-- In `OrderedRingBuffer` and `MovingWindow`:
-  - Support for integer indices is added.
-  - Add `count_covered` method to count the number of elements covered by the used time range.
-  - Add `fill_value` option to window method to impute missing values. By default missing values are imputed with `NaN`.
-- Add `at` method to `MovingWindow` to access a single element and use it in `__getitem__` magic to fully support single element access.
+- The configuration flag `resend_latest` can now be changed for channels owned by the `ChannelRegistry`.
 
-- The PowerDistributingActor now supports n:m relations between inverters and batteries.
+- New formula operators for calculating `consumption()` and `production()` were added.
 
-  This means that one or more inverters can be connected to one or more batteries.
+- The `ConnectionManager` now fetches microgrid metadata when connecting to the microgrid and exposes `microgrid_id` and `location` properties of the connected microgrid.
 
-- A `PowerManagingActor` implementation.
+  Users can access this information using `microgrid.connection_manager.get().microgrid_id` and `microgrid.connection_manager.get().location`.
```

Comment:
Also I think this got really hard to access, I wonder if we should provide something less cryptic, like `microgrid.microgrid_id()` and `microgrid.location()`. If we agree on it I will create a new issue.

Reply:
My only minor concern is those methods might return `None` which is not very suitable for the `microgrid` high level interface. Anyway I agree currently it is hard to access it and  we can create the issue and discuss it there